src/lib/components/Treebank/Sentence/Sentence.js

```diff
@@ -91,13 +105,15 @@ Sentence.propTypes = {
   callback: func,
   active: string,
   setActive: func,
+  highlighted: oneOfType([arrayOf(string), instanceOf(Set)]),
```

Comment:
Prop Types are a kind of type checking used by React. Ideally, we'd want to say it's either an array of strings or a set of strings, but there is no `setOf` type. `instanceOf(Set)` is, I think, the next best thing.